astropy/utils/system_info.py

```diff
@@ -0,0 +1,87 @@
+r"""
+A helper script to automate system info in bug reports.
+
+This can be run as
+```
+python -m astropy.utils.report_info
+```
+(note that the interpreter might be called python3 instead of python, depending
+on your system's details)
+
+This script should not require anything else than the standard library.
+"""
+
+__all__ = ["system_info"]
```

Comment:
Defining `__all__` makes this module look public, so there should be an explicit warning in the module docstring that the contents here are subject to change without notice.